lib/createWorkers.js

```diff
@@ -361,7 +408,7 @@ function createWorkers(handlers, config, options = {}) {
  * @param   {function} handler.handle the message handler
  * @param   {function} handler.validate the message validator
  * @param   {String} handler.routingKey the routingKey to bind to
- * @returns {*} -
+ * @returns {void}
```

Comment:
I prefer to use `@returns {undefined}` since `void` is not a type in JS, but whatevs

Reply:
seems that the difference between the two is that void doesn't have a return statement (search for void in https://eslint.org/docs/rules/valid-jsdoc)